File: MS_lifehealthservices/LHSAPI.Application/Client/Models/ClientProgressNotes.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace LHSAPI.Application.Client.Models
{
  public class ClientProgressNotes
  {
    public int Id { get; set; }

    public int ClientId { get; set; }

    public string PatientName { get; set; }

    public DateTime DateOfBirth { get; set; }

    public string MedicalRecordNo { get; set; }

    public string ScheduleFor { get; set; }

    public string AppointmentTo { get; set; }

    public string ForDescharge { get; set; }

    public string Other { get; set; }

    public DateTime ReviewDate { get; set; }

    public DateTime? SignedDate { get; set; }

    List<ProgressNotesList> ProgressNotesList { get; set; }
  }
}

File: MS_rosterfrontend/projects/employee-checkout/src/lib/view-models/client-progress-notes.ts
export interface ClientProgressNotes {
    id?: number;
    clientId?: number;
    patientName?: string;
    dateOfBirth?: string;
    medicalRecordNo?: string;
    scheduleFor?: string;
    appointmentTo?: string;
    forDescharge?: string;
    other?: string;
    reviewDate?: string;
    signedDate?: string;
}

export interface ClientProgressNotesFields {
    date?: Date;
    note9AMTo11AM?: string;
    note11AMTo1PM?: string;
    note1PMTo15PM?: string;
    note15PMTo17PM?: string;
    note17PMTo19PM?: string;
    note19PMTo21PM?: string;
    note21PMTo23PM?: string;
    note23PMTo1AM?: string;
    note1AMTo3AM?: string;
    note3AMTo5AM?: string;
    note5AMTo7AM?: string;
    note7AMTo9AM?: string;
    summary?: string;
    communityAccess?: string;
    culturalNeeds?: string;
    behaviourConcern?: string;
    medicationgiven?: string;
    anyFalls?: string;
    mobilitySafety?: string;
    nutritionalDetail?: string;
    exerciseDone?: string;
    otherInfo?: string;
    id?: number;
    clientId?: number;
    clientProgressNoteId?: number;
}
